io.github.DamnedAngel.msx-tile-forge.yaml

```diff
@@ -0,0 +1,265 @@
+app-id: io.github.DamnedAngel.msx-tile-forge
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+
+command: msxtileforge
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=xdg-documents
+
+modules:
+  # tkinter dependencies
+  - name: tkinter
+    buildsystem: simple
+    build-options:
+      env:
+        CFLAGS: "-I/app/include/python3.12 -I/app/include/python3.12/internal -I/app/include"
```

Comment:
use cflags to set these also the value python3.12 will break on runtime updates.

Reply:
```yaml
build-options:
  cflags:
```